src/assemble_workflow/bundle.py

```diff
@@ -135,7 +135,7 @@ def __get_min_dist(self, build_components: BuildComponents) -> Dist:
         logging.info(f"Copied min bundle to {min_dist_path}.")
         min_path = f"{self.build.filename}-{self.build.version}".replace("-SNAPSHOT", "")
         logging.info(f"Start creating distribution {self.build.distribution} for {self.min_bundle.name}.")
-        min_dist = Dists.create_dist(self.min_bundle.name, min_dist_path, min_path, self.build.distribution)
+        min_dist = Dists.create_dist(self.min_bundle.name, min_dist_path, min_path, self.build.__to_dict__())
```

Comment:
I think you misunderstood me. Here pass in `self.build`.

Reply:
fixed.